loginForm.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <link rel="manifest" href="manifest.json">
+      <link href="https://fonts.googleapis.com/css?family=Hanalei+Fill" rel="stylesheet">
+  </head>
+
+  <p class="header">Login Form</p>
+  <form>
+    <input id="username" type="text" placeholder="Username" /><br />
+    <input id="password" type="password" placeholder="Password" /><br />
+    <input type="submit" id="submit" />
+  </form>
+
+
+</html>
+
```

Comment:
Nit: Please add a title to the head and remove the blank space (line 16/17) (W3C HTML validator)

Reply:
Done, thanks!